eng/Versions.props

```diff
@@ -9,19 +9,16 @@
     <AspNetCoreMajorVersion>3</AspNetCoreMajorVersion>
     <AspNetCoreMinorVersion>2</AspNetCoreMinorVersion>
     <AspNetCorePatchVersion>0</AspNetCorePatchVersion>
-    <PreReleasePreviewNumber>5</PreReleasePreviewNumber>
+    <PreReleasePreviewNumber>1</PreReleasePreviewNumber>
     <!--
         When StabilizePackageVersion is set to 'true', this branch will produce stable outputs for 'Shipping' packages
     -->
     <StabilizePackageVersion Condition="'$(StabilizePackageVersion)' == ''">false</StabilizePackageVersion>
     <DotNetFinalVersionKind Condition="'$(StabilizePackageVersion)' == 'true'">release</DotNetFinalVersionKind>
     <IncludePreReleaseLabelInPackageVersion>true</IncludePreReleaseLabelInPackageVersion>
     <IncludePreReleaseLabelInPackageVersion Condition=" '$(DotNetFinalVersionKind)' == 'release' ">false</IncludePreReleaseLabelInPackageVersion>
-    <PreReleaseVersionLabel>preview$(PreReleasePreviewNumber)</PreReleaseVersionLabel>
-    <PreReleaseBrandingLabel>Preview $(PreReleasePreviewNumber)</PreReleaseBrandingLabel>
-    <!-- Blazor Client packages will not RTM with 3.1 -->
-    <BlazorClientPreReleasePreviewNumber>4</BlazorClientPreReleasePreviewNumber>
-    <BlazorClientPreReleaseVersionLabel>preview$(BlazorClientPreReleasePreviewNumber)</BlazorClientPreReleaseVersionLabel>
+    <PreReleaseVersionLabel>rc$(PreReleasePreviewNumber)</PreReleaseVersionLabel>
```

Comment:
Do we really want semver1 for all packages produced in this branch? That's how the preview packages were versions but was that intentional?

Either way, if you make this just
``` msbuild
<PreReleaseVersionIteration>1</PreReleaseVersionIteration>
<PreReleaseVersionLabel>rc</PreReleaseVersionLabel>
```
the SDK will create appropriate version suffixes depending on how `$(SemanticVersioningV1)` is set. That Boolean property defaults to empty i.e. this change would automatically switch the package versions to semver2 unless you also add
``` msbuild
<SemanticVersioningV1>true</SemanticVersioningV1>
```
Either way, I recommend using the Arcade SDK as it's intended to be used and making an explicit and visible choice about semantic versions.

Reply:
Does `PreReleaseVersionIteration` work with the Arcade version we have in the blazor-wasm branch? If so I can give it a try. 

AFAIK, our packages have build numbers in them. What specific Semver v2 feature are we missing?

Reply:
`$(PreReleaseVersionIteration)` should work fine.

> What specific Semver v2 feature are we missing?

Well, the period between `rc` and `1`. The two formats look different and we generally build packages with Semver v2 versions elsewhere. Note the difference between e.g. https://www.nuget.org/packages/Microsoft.AspNetCore.Authentication.JwtBearer/5.0.0-preview.2.20167.3 (Semver v2) and https://www.nuget.org/packages/Microsoft.AspNetCore.Components.DataAnnotations.Validation/3.2.0-preview4.20210.8 (Semver v1).

Note that Semver v2 is supported in most contexts but not all. I can't guarantee v2 is what you want for all of these packages.

Reply:
Spoke to @danroth27 offline about this. He thinks we should continue using SemVer v1 to keep with the tradition.

Reply:
I guess.

FYI @danroth27 these packages will stick out if we do the same thing in 5.0. Site Extensions packages are the only place where we use Semver v1 (due to unique Antares requirements).